docs/design-proposals/arch/daemonset.md

```diff
@@ -2,6 +2,8 @@
 
 Due to the nature Arktos being a multiple-tenanted system, DaemonSet is only allowed in system tenant; only cluster admin is permitted to create/delete/update DaemonSet resources.
 
+Multiple-TP also causes similar issue to Kube-proxy; changes to kube-proxy is covered in this design.
+
 ### What is not covered yet
```

Comment:
can we add feature mentioned in line 28 ( recover DS from failed TPs ) to the not covered in this doc ?

Reply:
will add it to no scope.